fuzzer/modules/sql_injection/strategies/create_table_strategy.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import requests
+import uuid
+
+from functools import partial
+
+from .lib.postgres import get_tables, drop_table, DBCONFIG
+
+
+def openapi_api_call_generator(base_url, attack_param):
+    # TODO: Implement this for reals
```

Comment:
This is where we need to integrate your OpenAPI parser in to generate API calls.